### ISSUE TITLE
Refactor "No Results" Message from the Toolkit Filter and Reuse it for the Projects Filter

### ISSUE THREAD

**Issue by chrismenke45** [hackforla/website#4903]
### Overview

The Projects Filter shifts to the center when no results are found by the filter. We need to add a "No Results" message so this shift does not occur.

### Details

The Toolkit page filter displays a "no results" message when no results are found by the filter. The code from `toolkit.js` will need to be 

- extracted into a new component/ element file
- refactored so that it can be reused on the projects and toolkit filters (or any future filter) 

Once the above is performed, the code can then be imported into the projects filter file (or any future filter file).

<details>
<summary>Current Functionality</summary>

![Screen Shot 2023-07-01 at 7 23 48 AM](https://github.com/hackforla/website/assets/86500980/caa6aa3e-8de9-4276-9e23-8d52f3d02251)
</details>

<details>
<summary>Desired Functionality</summary>

![Screen Shot 2023-07-01 at 7 25 15 AM](https://github.com/hackforla/website/assets/86500980/3535c413-a2ee-45a4-9918-78f72f22d996)
</details>

### Action Items
- [x] Extract the HTML code from `toolkit.js` into a new file. Create the new file in the `_includes` directory. 
  - [x] Style the HTML as needed to match the design in [Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?type=design&node-id=24097-286538). The "no results" message should look the same on any page. 
- [x] Extract the "no results" JS code from `toolkit.js` into a new file. Create the new file in the `elements` directory (path: `/assets/js/elements`).
- [x] Refactor the "no results" code so that it 
   - [x] functions similarly to the filter on the toolkit page when no results are found
   - [x] can be reused on the home, projects, and toolkit pages
- [x] Refactor the filter code on the home, projects, and toolkit pages as needed to get the "no results" code to work
- [x] Import/ include the code in the relevant filter files for the home, projects, and toolkit pages. 

### Resources/Instructions
- This issue was created for ER #4769. Reference the ER for more background and information. 
- [Write up of how to implement the 'no results" message on another page](https://github.com/hackforla/website/issues/4769#issuecomment-1592092127)
- [Write up of how to implement the projects filter on another page](https://github.com/hackforla/website/issues/3744#issuecomment-1496291746)
- Files to reference:
  - [Toolkit Page Filter JS Code](https://github.com/hackforla/website/blob/c536e346051b60957d46c3bcd62eef9118314645/assets/js/toolkit.js)
  - [Projects Page Filter JS Code](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js)
  - [Home Page HTML Code](https://github.com/hackforla/website/blob/gh-pages/pages/index.html)
  - [Projects Page HTML Code](https://github.com/hackforla/website/blob/gh-pages/_includes/current-projects.html)
  - [Toolkits Page HTML](https://github.com/hackforla/website/blob/gh-pages/pages/toolkit.html)
- [Figma Design of "No Results" Message](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?type=design&node-id=24097-286538)

**Comment by chrismenke45** [comment]
This is an issue for ER #4769

**Comment by ExperimentsInHonesty** [comment]
@chrismenke45 It seems like this issue is to copy the code from the toolkit page and make some changes to it.  

What we should be doing is making an issue to refactor the code so that it can work for the toolkit page (and any other page) by calling a file that has the specific page variables in it. (filter names, messages, etc.).  

Once this refactor is done, then this issue can be revised to setup the file for this page with its variables in it.  And then call the script with the variable file to deliver a customized result to this page.

**Comment by chrismenke45** [comment]
@ExperimentsInHonesty Sounds good. I'll update it to reflect that, I am going to split it into two issues:
1. Update message on Toolkit page to be reusable component
2. Implement reusable component on Projects filter

**Comment by chrismenke45** [comment]
I am taking a break from HFLA and will not be able to complete writing this issue.

**Comment by Adastros** [comment]
Reopening this issue for ER #4769 and moving it back into the new issue approval column

**Comment by bphan002** [comment]
Availability: August 15th Tuesday
ETA: August 19th Saturday

**Comment by bphan002** [comment]
Updated ETA: August 23rd

Underestimated complexity
